distribution/pom.xml

```diff
@@ -169,6 +169,24 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
+                    <execution>
```

Comment:
Instead of a antrun execution, can you use maven download plugin with `unpack=true` option to do this.
We are already doing similar execution with download plugin in https://github.com/nihlaakram/product-microgateway/blob/b32387258e236161de282934352d961e0fd92464/distribution/pom.xml#L148

Reply:
This is the convention followed for all products.